data-prepper-plugins/opensearch/src/integrationTest/java/org/opensearch/dataprepper/plugins/sink/opensearch/OpenSearchSinkIT.java

```diff
@@ -113,7 +128,7 @@ public void cleanOpenSearch() throws Exception {
   @Test
   public void testInstantiateSinkRawSpanDefault() throws IOException {
     final PluginSetting pluginSetting = generatePluginSetting(IndexType.TRACE_ANALYTICS_RAW.getValue(), null, null);
-    OpenSearchSink sink = new OpenSearchSink(pluginSetting, pluginFactory);
+    OpenSearchSink sink = new OpenSearchSink(pluginSetting, pluginFactory, acknowledgementSetManager);
```

Comment:
This is one of the reasons I recommend using a `createObjectUnderTest()` method.

Not needed for this PR though.

Reply:
I remembered createObjectUnderTest() when I was making this change :-)

Reply:
I have Incorporated this change in this PR